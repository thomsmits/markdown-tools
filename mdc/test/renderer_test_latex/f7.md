# Chapter
## Slide Footnote

Text using a footnote[^1] and another one[^label]. Quote[^2].

[^1]: Footnote with number.
[^2]: Footnote with "quote".
[^label]: Footnote with label.

