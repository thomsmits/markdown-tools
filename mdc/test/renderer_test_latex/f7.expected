# Chapter
## Slide Footnote

Text using a footnote\footnote{Footnote with number.} and another one\footnote{Footnote with label.}. Quote\footnote{Footnote with \enquote{quote}.}.\vspace{0.1mm}
